scapy/contrib/automotive/uds.py

```diff
@@ -132,6 +135,7 @@ class UDS_DSC(Packet):
 
     @staticmethod
     def get_log(pkt):
+        # type: (UDS_DSC) -> Tuple[str, Any]
```

Comment:
Can we do better than using Any here?

Reply:
I'm currently working on a huge refactoring of my code which uses this log functions. Also this scanner code on the bottom of this file will move in a new file. After I've this code ready, I would focus on typing of this layer. Hope that works for now.